src/main/kotlin/step3/domain/GameResult.kt

```diff
@@ -2,8 +2,14 @@ package step3.domain
 
 class GameResult(
     private val initialPositions: CarPositions,
-    private val positionsResult: MutableList<CarPositions> = mutableListOf()
+    positionsResult: MutableList<CarPositions> = mutableListOf()
 ) {
+    private val positionsResult: MutableList<CarPositions>
+
+    init {
+        this.positionsResult = positionsResult.toMutableList()
+    }
```

Comment:
테스트가 통과하도록 잘 구현해주셨네요. 👍 
지금도 좋지만, 가급적이면 외부에서 보이는 타입은 Immutable(불변)으로 만드는 것이 좋습니다.

```suggestion
    positionsResult: List<CarPositions> = emptyList()
) {
    private val positionsResult = positionsResult.toMutableList()
```